app/views/articles/index.html.erb

```diff
@@ -47,7 +47,7 @@
         <div class="flex-1 flex items-center justify-between">
           <h2 class="fs-l fw-heavy">Posts</h2>
           <nav class="crayons-tabs hidden s:flex">
-            <a href="<%= list_path %>" class="crayons-tabs__item <%= "crayons-tabs__item--current" if %w[week month year infinity latest].exclude?(params[:timeframe]) %>">Feed</a>
+            <a href="<%= list_path %>/" class="crayons-tabs__item <%= "crayons-tabs__item--current" if %w[week month year infinity latest].exclude?(params[:timeframe]) %>">Feed</a>
```

Comment:
@benhalpern Thinking about this more...maybe `list_path` can just return `"/"` if there's no tag and then we don't need to worry about an empty path since the helper can handle it internally...